module1/homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,26 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence==0){
+        return 0;
+    }
+    if(sequence==1 || sequence==2){
+        return 1;
+    }
+    int last = 1, conteiner , score = 1; 
+    for(int i = 0;i<sequence-2;i++){
+        
+        conteiner=score;
+        score +=last;
+        last = conteiner;
+
+    }
+    return score;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence==0)return 0;
+    if(sequence <= 2)return 1;
```

Comment:
inconsistent formatting. Please apply spaces before and after operators like ==, +, -
```suggestion
    if (sequence == 0) return 0;
    if (sequence <= 2) return 1;
```